controller/users.go

```diff
@@ -596,6 +597,35 @@ func (c *UsersController) Update(ctx *app.UpdateUsersContext) error {
 			}
 		}
 
+		updatedFeatureLevel := ctx.Payload.Data.Attributes.FeatureLevel
```

Comment:
There's a lot of feature-toggle specific logic here - wondering if we should move it in a separate function to avoid the already huge controller ( blame me ) from grown bigger :)

Reply:
ok, I moved it in a separate, dedicated method of the controller: https://github.com/fabric8-services/fabric8-auth/pull/238/commits/fbeb03643b66158c8e8ace5d14d4dfb065c58c5a